lib/winston/transports/file.js

```diff
@@ -532,6 +533,32 @@ module.exports = class File extends TransportStream {
     }
   }
 
+  /**
+   * To fix issue https://github.com/winstonjs/winston/issues/228
+   * @private
+   * @param {function} cb Delaying the 'finish' event until callback is called
+   * @returns {undefined}
+   */
+  _waitStreamsCreatedButNotPiped(cb) {
+    if (this._destOpened) {
+      return cb();
```

Comment:
Potentially violates the  @returns {undefined} docs as cb could return anything?